src/actions/task-definitions.js

```diff
@@ -198,6 +198,22 @@ tasks.changeToPluginCell = new UserTask({
   callback() { dispatcher.changeCellType('plugin') },
 })
 
+tasks.includeCellInRunAll = new UserTask({
+  title: 'Include Cell During Run All',
+  // keybindings: ['l'],
+  // displayKeybinding: 'L',
+  // keybindingPrecondition: isCommandMode,
+  callback() { dispatcher.setCellIncludeInRunAll(true) },
+})
+
+tasks.skipCellInRunAll = new UserTask({
```

Comment:
philosophical question - should this be `skipCellInRunAll` or should it be `skipCellDuringEvaluation`? If I have selected a cell I'm skipping and attempt to evaluate it, should it evaluate? I'd argue probably not. If I want to not have a cell evaluate when I click "run all cells", I don't want to be in the position to accidentally evaluate it, either.

OR, should this be `disableCell`, which makes it 1. uneditable, and 2. doesn't run in any context. This option feels the most consistent and should be clear to users that a disabled cell doesn't run in any context, nor is it editable until it is enabled.